tensorflow/lite/nnapi/NeuralNetworksShim.h

```diff
@@ -143,6 +143,7 @@ enum {
   ANEURALNETWORKS_STRIDED_SLICE = 35,
   ANEURALNETWORKS_SUB = 36,
   ANEURALNETWORKS_TRANSPOSE = 37,
+  ANEURALNETWORKS_CEIL = 38,
```

Comment:
This code isn't in Android so I don't think we can land it. Can we factor out the NNAPI additions separately?